src/clients/AcrossHubPoolClient.ts

```diff
@@ -511,7 +511,7 @@ export class HubPoolClient {
   }
 
   async update(eventsToQuery?: HubPoolEvent[]): Promise<void> {
-    eventsToQuery ??= Object.keys(this.hubPoolEventFilters()) as HubPoolEvent[]; // Query all events by default.
+    eventsToQuery = eventsToQuery ?? (Object.keys(this.hubPoolEventFilters()) as HubPoolEvent[]); // Query all events by default.
```

Comment:
Change `??=` to ```eventsToQuery = eventsToQuery ?? ...```;

Reply:
Is this needed to avoid some tsc or lint issue as a result of the type assertion on the ??= case?